src/components/FormSubmit/index.tsx

```diff
@@ -1,23 +1,21 @@
-import lodashGet from 'lodash/get';
-import React, {useEffect} from 'react';
-import {View} from 'react-native';
+import React, {KeyboardEvent as ReactKeyboardEvent, useEffect} from 'react';
+import {AccessibilityRole, View} from 'react-native';
 import * as ComponentUtils from '@libs/ComponentUtils';
 import isEnterWhileComposition from '@libs/KeyboardShortcut/isEnterWhileComposition';
 import CONST from '@src/CONST';
-import * as formSubmitPropTypes from './formSubmitPropTypes';
+import {FormSubmitProps, FormSubmitRef} from './types';
 
-function FormSubmit({innerRef, children, onSubmit, style}) {
+function FormSubmit({children, onSubmit, style}: FormSubmitProps, ref: FormSubmitRef) {
     /**
      * Calls the submit callback when ENTER is pressed on a form element.
-     * @param {Object} event
      */
-    const submitForm = (event) => {
+    const submitForm = (event: ReactKeyboardEvent) => {
         // ENTER is pressed with modifier key or during text composition, do not submit the form
-        if (event.shiftKey || event.key !== CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey || isEnterWhileComposition(event)) {
+        if (event.shiftKey || event.key !== CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey || isEnterWhileComposition(event as unknown as KeyboardEvent)) {
             return;
         }
 
-        const tagName = lodashGet(event, 'target.tagName', '');
+        const tagName = (event.target as HTMLElement)?.tagName ?? '';
```

Comment:
does it make sense to create a variable asserting 
```
const eventTarget = event.target as HTMLElement;
```
and then use that one?

Reply:
updated